quesma/queryparser/query_parser.go

```diff
@@ -427,7 +427,10 @@ func (cw *ClickhouseQueryTranslator) iterateListOrDictAndParse(queryMaps interfa
 		return cw.parseQueryMapArray(queryMapsTyped)
 	case QueryMap:
 		simpleQuery := cw.parseQueryMap(queryMapsTyped)
-		return []model.Expr{simpleQuery.WhereClause}, simpleQuery.CanParse
+		if simpleQuery.WhereClause != nil {
+			return []model.Expr{simpleQuery.WhereClause}, simpleQuery.CanParse
+		}
+		return []model.Expr{}, simpleQuery.CanParse
```

Comment:
Not sure why but after my fix a few tests started panicking (this `WhereClause` was a `null`, and some highlighter visitor panicked when it saw this `null`). I think this change makes sense